helm/cloudformation-operator/Chart.yaml

```diff
@@ -1,14 +0,0 @@
-apiVersion: v1
-appVersion: "v0.6.0"
-description: A Helm chart for Kubernetes operator for managing CloudFormation stacks
-name: cloudformation-operator
-version: 0.1.0
-home: https://github.com/linki/cloudformation-operator
-keywords:
-  - AWS
-sources:
-  - https://github.com/linki/cloudformation-operator
-maintainers:
-  - name: mbalazs90
-    email: xpbazsi@gmail.com
-engine: gotpl
```

Comment:
Let's keep the helm chart. It might be outdated but it would be sad to lose all the work that was done for it.

Reply:
Okay, makes sense. Do you care if the same directory? I can go dig up what it was, but then catch it up for this version of what the sdk+kustomize generates, parameterize it and put it back in.

Reply:
Done!

Reply:
Awesome. Thank you!